examples/routers/checkbox/demo/controller.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <div>
+    <p style="margin-bottom: 20px;">
+      <ant-checkbox v-model="check" :disabled="disable">Checkbox</ant-checkbox>
+    </p>
+    <p>
+      <ant-button @click="toggleCheck">{{ checkText }}</ant-button>
```

Comment:
1. Button add attribute `type="primary" size=small`, follow line should be added too.
2. Don't forget update in `/checkbox/demo`.